op-enable.go

```diff
@@ -356,3 +357,22 @@ func getArgs(s DockerHandlerSettings, dd driver.DistroDriver) string {
 
 	return strings.Join(args, " ")
 }
+
+
+func getDockerUrls(env string) (string, string) {
+	urlMap := map[string]struct{dockerUrl, composeUrl string}{
+		"AzureChinaCloud": {
+			// This script is synced with https://get.docker.com daily, with the change that apt_url and yum_url point to the mirror in China
```

Comment:
if you're going to keep doing the centos/redhat modifications, can you please keep the comment noting that?